ui-meta/workstation.yml

```diff
@@ -53,6 +53,12 @@ usage_template: |-
 
   Monitoring dashboards for the workstation are available using the [Monitoring service]({{ monitoring.url }}).
 
+  Note that this user storage (/home/azimuth) is EPHEMERAL and is removed on platform deletion, but is persistent
+  through platform patches. All system level changes outside of the home directory will be lost on a platform patch.
+
+  When configured by the Azimuth operator, an additional filesystem is mounted at /project. This filesystem persists
+  after the platform is deleted and is shared with other platforms in the tenancy.
+
```

Comment:
I don't think this makes sense in the context of the whole description - for instance, the /project share is already mentioned above. We probably need to review the whole description holistically.